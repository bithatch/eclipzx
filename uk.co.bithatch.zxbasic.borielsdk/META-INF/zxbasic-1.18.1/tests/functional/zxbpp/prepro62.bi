
#define q z
#define z(a, b) (a + 1, b + 2)

z(1, 2)
q(3, 4)
