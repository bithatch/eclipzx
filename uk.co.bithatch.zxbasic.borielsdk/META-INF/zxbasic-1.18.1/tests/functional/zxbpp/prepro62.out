#line 1 "prepro62.bi"

#line 3 "prepro62.bi"
#line 4 "prepro62.bi"

(1 + 1, 2 + 2)
(3 + 1, 4 + 2)
